pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -1283,17 +1283,13 @@ public boolean alreadyRelease(){
     }
 
     /**
-     * Changes the idle-state of the connection to #{@link IdleState#IDLE_MARKED}, This method only changes this
-     * connection from the #{@link IdleState#USING} state to the #{@link IdleState#IDLE_MARKED} state. if the
-     * idle-status is successfully changed, "idleMarkTime" is changed to current time.
-     * @return Whether change idle-stat to #{@link IdleState#IDLE_MARKED} success.
+     * Try to transform the state of the connection to #{@link IdleState#IDLE_MARKED}, state should only be transformed to #{@link IdleState#IDLE_MARKED} from state  #{@link IdleState#USING}. if the
+     * state is successfully transformed, "idleMarkTime" will be  assigned to current time.
```

Comment:
Hi @gaozhangmin 
src/main/java/org/apache/pulsar/client/impl/ClientCnx.java:[1286] (sizes) LineLength: Line is longer than 120 characters